tests/src/test/java/alluxio/server/ft/MasterFaultToleranceIntegrationTest.java

```diff
@@ -278,11 +278,12 @@ public void failoverWorkerRegister() throws Exception {
     // Stop the default cluster.
     after();
 
-    // Create a new cluster, with no workers initially
-    final MultiMasterLocalAlluxioCluster cluster = new MultiMasterLocalAlluxioCluster(2, 0);
-    cluster.initConfiguration();
-    cluster.start();
+    MultiMasterLocalAlluxioCluster cluster = null;
     try {
+      // Create a new cluster, with no workers initially
+      cluster = new MultiMasterLocalAlluxioCluster(2, 0);
```

Comment:
maybe we in the future can make MultiMasterAlluxioCluster implement the `AutoCloseable` interface so that we can use the `try-with-resources` syntax

Reply:
I realized most tests are using `before` and `after` methods for setting up and tearing clusters, which makes using try statements a bigger change.